WordPressAuthenticatorTests/Credentials/CredentialsTests.swift

```diff
@@ -0,0 +1,127 @@
+                                                  password: password,
+                                                  xmlrpc: xmlrpc,
+                                                  options: [:])
+        
+        XCTAssertEqual(wporgcredentials.username, username)
+        XCTAssertEqual(wporgcredentials.password, password)
+        XCTAssertEqual(wporgcredentials.xmlrpc, xmlrpc)
+    }
+    
+    func testWordPressOrgCredentialsEquatable() {
+        let lhs = WordPressOrgCredentials(username: username,
+                                          password: password,
+                                          xmlrpc: xmlrpc,
+                                          options: [:])
+        
+        let rhs = WordPressOrgCredentials(username: username,
+                                          password: password,
+                                          xmlrpc: xmlrpc,
+                                          options: [:])
+        
+        XCTAssertTrue(lhs == rhs)
+    }
+    
+    func testWordPressOrgCredentialsNotEquatable() {
+        let lhs = WordPressOrgCredentials(username: username,
+                                          password: password,
+                                          xmlrpc: xmlrpc,
+                                          options: [:])
+        
+        let rhs = WordPressOrgCredentials(username: "username5678",
+                                          password: password,
+                                          xmlrpc: xmlrpc,
+                                          options: [:])
+        
+        XCTAssertFalse(lhs == rhs)
+    }
+    
+    func testAuthenticatorCredentialsInit() {
+        let wporgCredentials = WordPressOrgCredentials(username: username,
+                                                       password: password,
+                                                       xmlrpc: xmlrpc,
+                                                       options: [:])
+        let wpcomCredentials = WordPressComCredentials(authToken: token,
+                                                       isJetpackLogin: false,
+                                                       multifactor: false,
+                                                       siteURL: siteURL)
+        let authenticatorCredentials = AuthenticatorCredentials(wpcom: wpcomCredentials,
+                                                                wporg: wporgCredentials)
+        
+        
+        XCTAssertEqual(authenticatorCredentials.wpcom!.authToken, token)
+        XCTAssertEqual(authenticatorCredentials.wpcom!.isJetpackLogin, false)
+        XCTAssertEqual(authenticatorCredentials.wpcom!.multifactor, false)
+        XCTAssertEqual(authenticatorCredentials.wpcom!.siteURL, siteURL)
+        XCTAssertEqual(authenticatorCredentials.wporg!.username, username)
+        XCTAssertEqual(authenticatorCredentials.wporg!.password, password)
+        XCTAssertEqual(authenticatorCredentials.wporg!.xmlrpc, xmlrpc)
```

Comment:
Does this compile?

```suggestion
        XCTAssertEqual(authenticatorCredentials.wporg?.xmlrpc, xmlrpc)
```

In the same spirit of an earlier comment, preventing crashes in the tests will help in the case in which they'll fail by providing clearer failure information.

Reply:
Yes it does :)  Updated to avoid the crashing